notifications-sample/src/extension.ts

```diff
@@ -0,0 +1,63 @@
+import * as vscode from 'vscode';
+
+export function activate(context: vscode.ExtensionContext) {
+
+	// Simple notifications
+	let showInfoNotification = vscode.commands.registerCommand('notifications-sample.showInfo', () => {
+		vscode.window.showInformationMessage('Info Notification');
+	});
+
+	let showInfoNotificationAsModal = vscode.commands.registerCommand('notifications-sample.showInfoAsModal', () => {
+		vscode.window.showInformationMessage('Info Notification As Modal', { modal: true });
+	});
+
+	let showWarningNotification = vscode.commands.registerCommand('notifications-sample.showWarning', () => {
+		vscode.window.showWarningMessage('Warning Notification');
+	});
+
+	let showErrorNotification = vscode.commands.registerCommand('notifications-sample.showError', () => {
+		vscode.window.showErrorMessage('Error Notification');
+	});
+
+	// Notifcation with actions
+	let showWarningNotificationWithActions = vscode.commands.registerCommand('notifications-sample.showWarningWithActions', () => {
```

Comment:
These are not being reassigned so could be const

Reply:
For whatever reason this command throws a modal dialog when the notification is visible and you run the command again

Reply:
The same dialog appears when trying to clear the notification center and this toast is visible

Reply:
Fixed